src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -38,15 +46,21 @@ public static IDisposable Open(TopLevel root)
             {
                 window = new MainWindow
                 {
-                    Width = 1024,
-                    Height = 512,
+                    Width = options.Size.Width,
+                    Height = options.Size.Height,
                     Root = root,
                 };
 
                 window.Closed += DevToolsClosed;
+                window.ViewModel.ShouldVisualizeDirtyRects = options.ShouldVisualizeDirtyRects;
+                window.ViewModel.ShouldVisualizeMarginPadding = options.ShouldVisualizeMarginPadding;
+                window.ViewModel.Console.IsVisible = options.ShowConsole;
+                window.ViewModel.ShowFpsOverlay = options.ShowFpsOverlay;
```

Comment:
Same with this: `window.Renderer.DrawFps`.

Reply:
I know they are already in the menu, but it's boring to set every FPS or DirtyRects , or whether or not to view the console.